indico/web/flask/app.py

```diff
@@ -202,6 +202,7 @@ def setup_jinja(app):
     app.add_template_global(render_field, '_render_field')
     app.add_template_global(iter_form_fields, '_iter_form_fields')
     app.add_template_global(format_currency)
+    app.add_template_global(date_time_util.format_interval)
```

Comment:
Why not make it a filter like the other date time utils? (btw if you do that please remove the incorrect "(indico functions returning UTF8)" comment)

Reply:
Can I have a filter with two arguments?

Reply:
yes, `foo|bar(x, ...)` basically calls `bar(foo, x, ...)`